dashboard/config/initializers/devise.rb

```diff
@@ -286,6 +286,22 @@
   # with a log in with facebook button)
   config.omniauth :clever, CDO.dashboard_clever_key, CDO.dashboard_clever_secret, provider_ignores_state: true
 
+  config.omniauth :openid_connect, {
```

Comment:
Just to make sure I understand this, this will config an `openid_connect` provider, configured for The School Project endpoint? Can we create multiple `openid_connect` providers if we have other OpenId-based products to connect to?